console/src/lib/tasks/Checkbox.svelte

```diff
@@ -16,6 +17,15 @@
 		checked: param.default || false
 	};
 
+	onMount(() => {
+		const urlParams = new URLSearchParams(window.location.search);
+		for (const [key, value] of urlParams) {
+			if (key === param.name) {
+				fieldValue = value === 'true';
+			}
+		}
```

Comment:
There's actually a couple of nice ways to get the param directly ([svelte kit docs](https://kit.svelte.dev/docs/web-standards#url-apis) give more info)

but instead of the `for` loop, you could:

```javascript
fieldValue = $page.url.searchParams.get(param.name) === 'true';
```

Other param types can use similar logic for fetching params. If the param doesn't exist, the value returned by `get` will be `null`.

At the top of the file you'll need to import `$page`

```
<script>
    import { page } from '$app/stores';
</script>
```